website/blog/2021-04-12-funding-update.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "Babel Funding Update"
+author: Babel Core Team
+date: 2021-04-12 0:00:00
+categories: announcements
+share_text: "Babel Funding Update"
+---
+
+In March 2018 [Henry](https://twitter.com/left_pad) started an [experiment](https://www.henryzoo.com/in-pursuit-of-open-source-part-1/): would it be possible to work full-time on Babel, entirely sustained by donations from Babel users?
+
+In November 2019, after successfully guaranteeing a salary to a maintainer for over a year, we [expanded our goal](https://babeljs.io/blog/2019/11/08/babels-funding-plans): would it also be possible to support three additional part-time maintainers ([Jùnliàng](https://twitter.com/JLHwung), [Kai](https://twitter.com/kai_cataldo) and [Nicolò](https://twitter.com/NicoloRibaudo))?
+
+Nicolò, Jùnliàng and Henry are still paid to work on Babel, and together with them there is a talented team that helps after or during their day-to-day job.
+
+<!-- truncate -->
+
+## Looking back
+
+After a few months being paid to work on Babel, Henry started receiving $11,000 USD per month. The goal was to establish the number as a baseline salary for working full-time on an open source project that seemed reasonable given the donations at the time.
+
+When we added three new paid maintainers, we couldn't afford to pay everyone the full $11,000 USD rate: Jùnliàng, Nicolò and Kai started at $2,000 USD per month, with the idea of increasing that amount when we would have received more donations.
+
+It has been more than one year since then and in the meanwhile Kai stepped down, but we still didn't have a chance to reconsider how much everyone is being paid.
+
+We have a hard time justifying the pay differences, and there isn't an obvious reason anymore for which someone should be paid more than the other team members:
+
+- We want to focus on ourselves as _people_, not as employees of a hypothetical company. We don't want to track meticulously how many hours we work, or how productive we are.
+- It's not obvious what we would consider as "working time". Part of what we do is to collaborate with other projects, but what exactly is the line between something done for Babel and something done as a personal contribution to the JavaScript ecosystem?
+- It's not possible to exactly measure how much we "produce": not every commit takes the same effort, not every task has the same impact, not every experiment has a direct result.
```

Comment:
Ideally, a roadmap with expected engineering outcomes would be generated even if those outcomes resulted in a decision to not do anything further. With a roadmap one can drive accountability of work and measure progress. 

While I agree this work is hard, through having clear and measurable outcomes. One can then justify funding and further investments. Hypothetical company or not the revenue from donations and Goodwill of the community is something that should be treated proportional to the position of this project in the open source funding ecosystem.

Reply:
Thanks for the comments Sam! Agreed on the roadmap, we had one a while back (https://babeljs.io/docs/en/roadmap) and made another one that we haven't posted yet. We're still writing up some changes, but glad to hear your thoughts.